checkstyle/suppressions.xml

```diff
@@ -9,11 +9,14 @@
     <suppress checks="ParameterName"
               files="ErrorMessage.java"/>
 
+    <suppress checks="ParameterNumber"
+              files="SchemaRegistryCoordinator.java"/>
```

Comment:
I'm open to suggestions about how to fix these if you see a way, but they are mostly due to these classes being integration points and are due to the way the classes for the new consumer in Kafka are written.